windows/wintools_uninstall_verify/wintools_uninstall_verify.yml

```diff
@@ -61,7 +60,7 @@
               vars:
                 vswitch_name: "{{ new_device_vswitch }}"
           when:
-            - guest_os_with_inbox_drivers | bool
+            - guest_os_with_inbox_drivers
```

Comment:
This when can be moved before block. And I saw there're some include tasks having names some not. As it is a big amount of tasks names, it's up to you whether to add them in this fix.

Reply:
I'll add task names in the files affected in this merge request firstly. Thanks.